grails-app/controllers/au/org/ala/ecodata/CommentController.groovy

```diff
@@ -0,0 +1,161 @@
+package au.org.ala.ecodata
+
+import grails.converters.JSON
+import groovy.json.JsonSlurper
+
+class CommentController {
+    def outputService
+
+    def ignores = ["action","controller"]
+
+    def list() {
+        List comments = []
```

Comment:
using int instead of def/string will throw an exception for me. this is since params.start value is a string even though the value can be converted to an integer.